Format configuration options in readme

Hubbitus opened the pull request:
Just for readability

Hubbitus committed:
Format configuration options in readme

igor-suhorukov reviewed (changes requested):
Thank you for update. Could you please revert string 
"config:file/http" AS IS. There are two hardcoded option to fetch  configuration from: "config:file" or "config:http"

Hubbitus committed:
org.aspectj.weaver.loadtime.configuration only "config:file" or "config:http"

Changes by review comment https://github.com/igor-suhorukov/aspectj-scripting/pull/4#pullrequestreview-25117366

Hubbitus commented:
Sure. Sorry for misunderstanding.

igor-suhorukov commented:
Thank you for markdown formatting!)
